packages/dds/tree/src/test/shared-tree/editing.spec.ts

```diff
@@ -143,13 +145,13 @@ describe("Editing", () => {
 
 			const tree2 = tree1.branch();
 
-			const fooArrayPath: UpPath = {
+			const fooArrayPath: NormalizedUpPath = {
```

Comment:
I thought that NormalizedUpPath requires detachedNodeId to be explicitly defined, why's that not the case here?

Reply:
This is a case where the `NormalizedUpPathInterior` type is being used. Since that type represents an interior node, there's no way it could be a detached root. That type therefore doesn't require a `detachedNodeId` (in fact it will prevent setting anything but `undefined` to it)